src/Power.cpp

```diff
@@ -212,19 +229,16 @@ class AnalogBatteryLevel : public HasBatteryLevel
 
 #ifndef BAT_MEASURE_ADC_UNIT // ADC1
 #ifdef ADC_CTRL
-        if (heltec_version == 5) {
-            pinMode(ADC_CTRL, OUTPUT);
-            digitalWrite(ADC_CTRL, HIGH);
-            delay(10);
-        }
+        pinMode(ADC_CTRL, OUTPUT);
+        digitalWrite(ADC_CTRL, HIGH);
+        delay(10);
 #endif
         for (int i = 0; i < BATTERY_SENSE_SAMPLES; i++) {
             raw += adc1_get_raw(adc_channel);
+            delayMicroseconds(100);
```

Comment:
What problem is this solving? IDK that we should be delaying in something that runs as often as this task runs.

Reply:
none, this could be removed, i just gave a look at adc1_get_raw and it's waiting on it's own for the sampling. a delay may help with noise mitigation if the board has a LPF but i just wrote it as an habit.
I also think 10ms for the ADC_CTRL it's way too much but i hadn't time to give a peek with the oscilloscope.